pkg/spec/populators_test.go

```diff
@@ -209,19 +209,6 @@ func TestPopulateServicePortNames(t *testing.T) {
 		outputServicePorts []api_v1.ServicePort
 	}{
 		{
-			name: "Passing only one servicePort, no population should happen",
```

Comment:
This teste should stay.
Just update `outputServicePorts` so it checks if `Name` was filled.

Reply:
done!